addon/components/modal-view.js

```diff
@@ -4,11 +4,13 @@ import layout from '../templates/components/modal-view';
 export default Ember.Component.extend({
   layout,
   classNames: ['modal', 'fade'],
+  attributeBindings: ['tabindex'],
+  tabindex: -1,
 
   didInsertElement() {
-    this.$(this.element).modal({keyboard: false, backdrop: 'static'});
+    this.$().modal({ keyboard: true, backdrop: 'static' });
+    this.$().on('hide.bs.modal', () => {
+      this.triggerAction({ action: 'closeModal' });
+    });
   },
-  willDestroyElement() {
-    this.$(this.element).modal('hide');
```

Comment:
you can not delete it otherwise, if you change to another page the modal won't hide.

May be just add a conditional  to ensure the existence of the elemeny